docs/en/observability/synthetics-private-location.asciidoc

```diff
@@ -1,5 +1,24 @@
 [[synthetics-private-location]]
-= Add a private location
+= Monitor a private endpoint
+
+To monitor resources on private networks use {{agent}} to create a Private Location, or allow access to your private endpoints to the Synthetics Service.
+Private Locations via Elastic Agent require only outbound connections from your network, while allowing access to the Synthetics Service requires allowing
+inbound access, thus posing an additional risk that users must assess.
+
+[discrete]
+[[monitor-via-access-control]]
+= Allow the synthetics service access to your private network
```

Comment:
Choose one style:

* Synthetics Service
* synthetics service
* Synthetics service

```suggestion
= Allow the Synthetics service access to your private network
```

Reply:
I think it should be "Synthetics Service". WDYT? If you're OK with that I'll add it here. Should it become a variable?

Reply:
The _proper_name is `Elastic's global managed infrastructure` like we use [elsewhere](https://www.elastic.co/guide/en/observability/master/synthetics-private-location.html#synthetics-private-location-next), should probably use that for consistency